web/annotation_editor_layer_builder.css

```diff
@@ -107,6 +102,44 @@
   cursor: auto;
   max-width: 100%;
   max-height: 100%;
+  border: var(--unfocus-outline);
+
+  &.draggable.selectedEditor {
+    cursor: move;
+  }
+
+  &.selectedEditor {
+    border: var(--focus-outline);
+    outline: var(--focus-outline-around);
+
+    &::before {
+      /*
+        This is a workaround for the lack of support for stripes(...) (see
+        https://drafts.csswg.org/css-images-4/#stripes).
+        The outline should be composed of 1px white, 2px blue and 1px white.
+        This could be achieved in different ways:
+          - using a linear-gradient;
+          - using a box-shadow;
+          - using an outline on the selected element and an outline+border on
+            the ::before pseudo-element.
+        All these options lead to incorrect rendering likely due to rounding
+        issues.
+        That said it doesn't mean that the current is ideal, but it's the best
+        we could come up with for the moment.
+        One drawback of this approach is that we use a border on the selected
+        element which means that we must take care of it when positioning the
+        div in js (see AnnotationEditor._borderLineWidth in editor.js).
+      */
```

Comment:
Given the problems of implementing this in a "nicer" way, any chance that you could ask UI/UX for a slightly tweaked specification such that we could avoid jumping through the hoops of having to make changes to the JS code as well here (to improve maintainability of this)?

Reply:
The problem of having only one line is that it could be almost invisible on a blue image which is maybe not that frequent but in hcm, it could be for sure more frequent.
So the two surrounding lines are here to contrast with the inner one and to almost guarantee that the outline will be always visible, hence I think we really need to have them especially because we don't have the control on the background.
That said, I shew the new outline to few people and everybody was happy with this design.
If you've any ideas to have an 100% CSS solution, please tell me, I'd prefer having one.

Reply:
![image](https://github.com/mozilla/pdf.js/assets/5641725/b9d6ba5d-5da6-4abd-b3bb-c410c2dfa601)
This shows why having three contrasted lines helps to see the outline whatever the background is.